tests/chainer_tests/test_link.py

```diff
@@ -1383,9 +1411,11 @@ def test_copyparams(self):
         with c2.init_scope():
             c2.c1 = c1
             c2.l3 = l3
+            c2.x = chainer.Parameter(shape=2)
         l1.x.data.fill(0)
         l2.x.data.fill(1)
         l3.x.data.fill(2)
+        c2.x.data.fill(3)
```

Comment:
How about this check below: `numpy.testing.assert_array_equal(self.c2.x.data, c2.x.data)`